README.md

```diff
@@ -2,38 +2,21 @@
 
 A Wireshark plugin for the [Bitnomial Transfer Protocol (BTP)](https://bitnomial.com/docs/bitnomial-transfer-protocol/): a binary, low latency, direct market access, trading and market data protocol.
 
-## Requirements
+## Installation
 
-- CMake >= 3.5 (>= 3.7 on 32 bit Windows)
-- Python >= 3.4
-- Qt >= 5
+To install the BTP Wireshark plugin, copy `btp.lua` to the correct path for your operation system
 
+ - Windows users can copy the file to `%APPDATA%\Wireshark\plugins\` or `WIRESHARK\plugins\`, where WIRESHARK is their Wireshark installation location.
+ - Unix-like users can copy the file to `~/.local/lib/wireshark/plugins`
```

Comment:
You might wanna just say "macOS, Linux, and other unix-like" or something like that.

Reply:
I clarified instructions for macOS and Linux, and fixed the broken link to the wireshark manual